sdrgui/mainwindow.cpp

```diff
@@ -174,10 +174,12 @@ MainWindow::MainWindow(qtwebapp::LoggerWithFile *logger, const MainParser& parse
 	connect(&m_inputMessageQueue, SIGNAL(messageEnqueued()), this, SLOT(handleMessages()), Qt::QueuedConnection);
 
     connect(screen(), &QScreen::orientationChanged, this, &MainWindow::orientationChanged);
+    #if (QT_VERSION < QT_VERSION_CHECK(6, 0, 0))
     screen()->setOrientationUpdateMask(Qt::PortraitOrientation
                                       | Qt::LandscapeOrientation
                                       | Qt::InvertedPortraitOrientation
                                       | Qt::InvertedLandscapeOrientation);
+    #endif
```

Comment:
no #else ?

Reply:
No else because the function setOrientationUpdateMask is removed  sice qt 6 and the property comes by default set to all these four orientations so no need for calling it ,same apply to another comment of yours